tests/integration/redis-benchmark.tcl

```diff
@@ -126,6 +126,14 @@ start_server {tags {"benchmark network external:skip logreqres:skip"}} {
             r config set maxclients $original_maxclients
         }
 
+        test {benchmark: read last argument from stdin} {
+            set base_cmd [redisbenchmark $master_host $master_port "-x -n 10 set key"]
+            set cmd "printf arg | $base_cmd"
```

Comment:
now that it's not a binary string, we can use `echo` right?

Reply:
Yes. Is it preferred?

Reply:
i think it's much more common in bash and therefore a bit more readable. AFAIK printf is only used when complicated things are needed.

Reply:
One difference is that printf doesn't add a trailing newline. (Not sure if it matters in this case.)